flink-runtime-web/web-dashboard/package.json

```diff
@@ -9,72 +9,73 @@
     "lint": "eslint --cache src --ext .ts,.html && stylelint \"**/*.less\"",
     "lint:fix": "eslint --fix --cache src --ext .ts,.html && stylelint \"**/*.less\" --fix",
     "ci-check": "npm run lint && npm run build",
-    "proxy": "ng serve --proxy-config proxy.conf.json"
+    "proxy": "ng serve --proxy-config proxy.conf.json",
+    "prepare": "cd ../.. && husky install flink-runtime-web/web-dashboard/.husky",
+    "lint-staged": "lint-staged"
   },
-  "private": true,
+  "private": false,
```

Comment:
What's the reason to change this? We don't ever want to publish this as a package.

Reply:
Ok, I will revert it.